definitions/tools/custom_merge_sv_records.cwl

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "merges nearby DEL/DUP records within a certain window distance"
+
+baseCommand: ["/bin/bash", "run_merge.sh"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: DockerRequirement
+      dockerPull: "apaul7/analysis:1.0.0"
```

Comment:
Is this an image that we should promote to the `mgibio` namespace?

Reply:
I moved the script in the cwl and used a different docker image